core/tee/sub.mk

```diff
@@ -30,6 +30,7 @@ srcs-y += tee_svc_cryp.c
 srcs-y += tee_svc_storage.c
 srcs-$(CFG_RPMB_FS) += tee_rpmb_fs.c
 srcs-$(CFG_REE_FS) += tee_ree_fs.c
+srcs-$(CFG_REE_FS) += fs_htree.c
```

Comment:
I'll use `srcs-$(call cfg-one-enabled,CFG_REE_FS CFG_SQL_FS)` instead as that's what we're using for `tee_fs_rpc.c`, I suppose that's equivalent.